alot/widgets/colour_text.py

```diff
@@ -0,0 +1,42 @@
+import logging
```

Comment:
please add a licence header as in all other files. You are welcome to add your name in there of course

Reply:
Just added it